### DISCUSSION TITLE
知乎相关域名

### DISCUSSION THREAD

**Discussion by VoidGod0** [SagerNet/SagerNet#503] [Q&A]
为什么打开知乎sg会连接到appcloud2.in.zhihu.com，然后就一直在那failed to lookup ip ，然而其它代理软件如surfboard,clash则连的是appcloud2.zhihu.com。在sg开HTTP入站以后甚至刷不了知乎。

**Comment by mailshuxin** [comment]
我也发现这个问题

**Comment by zsj1zsj** [comment]
这个问题解决了吗？@VoidGod0 @mailshuxin

**Comment by zedifen** [comment]
Maybe bypassing the Zhihu app in the VPN will do.

**Reply by zsj1zsj** [reply]
Tried it and it indeed works, thank you. 

It's strange that when I configure Alibaba's DNS in the client, it can't resolve appcloud2.in.zhihu.com, and zhihu app crashed due to this. And it seems only zhihu app has this issue.
